test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DisasterRecovery/BackupTests.cs

```diff
@@ -18,11 +18,21 @@ namespace Microsoft.SqlTools.ServiceLayer.IntegrationTests.DisasterRecovery
 {
     public class BackupTests
     {
+        // Query format to create master key and certificate for backup encryption
+        private const string CreateCertificateQueryFormat = @"USE master;
+IF NOT EXISTS(SELECT * FROM sys.symmetric_keys WHERE symmetric_key_id = 101)
+CREATE MASTER KEY ENCRYPTION BY PASSWORD = 'Yukon900';
```

Comment:
>Yukon900 [](start = 44, length = 8)

we shouldn't put passwords in the source code..and it's highly likely this won't be the correct password in most environments.